popup.html

```diff
@@ -54,25 +54,25 @@
     <form class="p-2 mt-2">
       <div class="form-row m-0">
         <div class="col-md-6 mb-3 ml-auto mr-4" style="width: auto">
-          <label class="" for="minIMDBRating">Min IMDB Rating:</label>
-          <select class="form-control form-control-sm" id="minIMDBRating">
-            <option value="1" selected>All</option>
-            <option value="2">2</option>
-            <option value="3">3</option>
-            <option value="4">4</option>
-            <option value="5">5</option>
-            <option value="6">6</option>
-            <option value="7">7</option>
-            <option value="8">8</option>
-            <option value="9">9</option>
+          <label class="" for="minIMDBRating" style="color: #333333;">Min IMDB Rating:</label>
```

Comment:
I thought we decided to revert all color changes, but you have bunch of inline stylings here. Also we have bootstrap, so why do you need inline styles for white and black?